src/reader/ArticleReader.js

```diff
@@ -304,15 +309,27 @@ export default function ArticleReader({ api, teacherArticleID }) {
         <div id={"bottomRow"}>
           <ReviewVocabulary articleID={articleID} />
           <s.CombinedBox>
-            <h4> {strings.answeringMsg} </h4>
+            <p style={{padding: "0em 2em 0em 2em"}}> {strings.answeringMsg} </p>
             <LikeFeedBackBox
               api={api}
               articleID={articleID}
               articleInfo={articleInfo}
               setArticleInfo={setArticleInfo}
               source={UMR_SOURCE}
```

Comment:
one thing that's interesting is that here we have source, and below we don't. is there a good reason for this?

Reply:
Adding source to LikeFeedbackBox was a suggestion from Tiago to keep track of where the user has made a like. Difficultyfeedbackbox did not log the activity, so no source was passed. Logging and source is now added.